packages/toggle/tests/Toggle.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import * as emotion from "emotion";
+import { createSerializer } from "jest-emotion";
+import { shallow, render } from "enzyme";
+import toJson from "enzyme-to-json";
+
+import { Toggle } from "../";
+
+expect.addSnapshotSerializer(createSerializer(emotion));
+
+const primary = () => <div>primary</div>;
+const secondary = () => <div>secondary</div>;
+
+describe("Toggle", () => {
+  it("renders correctly", () => {
```

Comment:
Can you write a title that explains what you are testing here? Correctly is pretty vague.

Reply:
Yes